ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CoveragesPage.tsx

```diff
@@ -61,6 +66,13 @@ function CoveragesPage() {
         Cluster: getFilteredConfig(clusterSearchFilterConfig, ['Name']),
     };
 
+    const fetchProfilesStats = useCallback(() => getComplianceProfilesStats(), []);
```

Comment:
The `find` method below brings a question to mind. What are pro and con:
* `profilesStatsResponse` is independent of changes to `profileName` as here, if I understand correctly.
* `profileName` becomes hook dependency so stats update for major update that affects table results (versus not for minor updates like filtering and sorting) to reduce chance of inconsistency if compliance operators report new results to central since the original request? IF this, a comment seems helpful to explain why.

https://github.com/stackrox/stackrox/pull/11726/files#diff-7a780f8fe8b2b3eae294eae7e1f5a72c8a563db93349804d8b74e10d0bdd9f5dL68-L79

Reply:
Great callout, makes sense to fetch the latest results. Updated